poppy/transport/pecan/controllers/v1/ssl_certificates.py

```diff
@@ -72,11 +72,21 @@ def post(self):
             helpers.abort_with_message)
     )
     def delete(self, domain_name):
-        # For now we only support 'san' cert type
-        cert_type = pecan.request.GET.get('cert_type', 'san')
-
+        # Now, we are supporting 'san' and 'sni' cert types
         certificate_controller = \
             self._driver.manager.ssl_certificate_controller
+
+        try:
+            certs_info = certificate_controller.get_certs_info_by_domain(
+                domain_name=domain_name,
+                project_id=None)
```

Comment:
project_id has been forced to a default of None, which would mean that the cert created, would have to be created with no project_id, which is not the case.

Reply:
We could get the project_id from the headers information.